src/promptflow-devkit/promptflow/_sdk/_service/app.py

```diff
@@ -99,8 +101,19 @@ def create_app():
         handler = RotatingFileHandler(filename=log_file, maxBytes=1_000_000, backupCount=1)
         formatter = logging.Formatter("[%(asctime)s][%(name)s][%(levelname)s] - %(message)s")
         handler.setFormatter(formatter)
+
```

Comment:
Are these handler codes used in multiple places? If so, please package it as a function.

Reply:
It's used in pfs for now